src/main/java/com/cg/entity/Customer.java

```diff
@@ -27,20 +27,28 @@ public class Customer {
 	
 	@Column(name="positionNumber")
 	private int positionNumber;
+	
+	@Column(name="parkingDuration")
+	private int parkingDuration;
+	
+	@Column(name="paymentMethod")
+	private String paymentMethod;
 
 	public Customer() {
 		super();
 	}
 
 	public Customer(int customerId, String name, String vehicleNumber, int tokenNumber, String phoneNumber,
-			int positionNumber) {
+			int positionNumber, int parkingDuration, String paymentMethod) {
 		super();
 		this.customerId = customerId;
 		this.name = name;
 		this.vehicleNumber = vehicleNumber;
 		this.tokenNumber = tokenNumber;
 		this.phoneNumber = phoneNumber;
 		this.positionNumber = positionNumber;
+		this.parkingDuration = parkingDuration;
```

Comment:
Customer is first initialized when he meets security guard. At that time this constructor is called so we cannot include parking duration here in constructor parameter list. It will be better if we discuss again on this.